README.md

```diff
@@ -73,10 +92,12 @@ ul.style.textAlign = 'left'
 
 That's better.
 
-## `removeChild()`
+## Remove Elements from the DOM
 
 Now let's remove one of those `li`s.
 
+### `removeChild()`
+
 ``` javascript
 ul.removeChild(ul.querySelector('li:nth-child(2)'))
```

Comment:
I'm a bit wincey about `querySelector`. We've not taught it yet. We could add it to 

https://github.com/learn-co-curriculum/js-dom-and-events-the-dom-is-a-tree-readme

(and probably should?)

But let's suppose we did that. I would still not want to roll out pseudo-selectors at this stage....

Reply:
They have, however, seen pseudo-selectors in the CSS lessons.